modules/router-store/src/serializer.ts

```diff
@@ -1,10 +1,18 @@
 import { ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
 
-export abstract class RouterStateSerializer<T> {
+/**
+ * Simple router state.
+ * All custom router states / state serializers should have at least this property.
+ */
+export interface RouterState {
```

Comment:
This interface needs a different name as the `@angular/router` already exposes a `RouterState`. It also needs to be added to `src/index.ts`